tests/integration/test_lock.py

```diff
@@ -150,3 +150,66 @@ def test_complex_lock_deep_extras(PipenvInstance, pypi):
         assert c.return_code == 0
         assert 'tablib' in p.lockfile['default']
         assert 'pandas' in p.lockfile['default']
+
+
+@pytest.mark.skip_lock
+@pytest.mark.index
+@pytest.mark.needs_internet
+@pytest.mark.install  # private indexes need to be uncached for resolution
+def test_private_index_skip_lock(PipenvInstance):
+    # This uses the real PyPI since we need Internet to access the Git
+    # dependency anyway.
+    with PipenvInstance() as p:
+        with open(p.pipfile_path, 'w') as f:
+            contents = """
```

Comment:
since these two are the same, how do you feel about using a constant and/or a fixture:

```
@pytest.mark.install # private indexes need to be uncached for resolution
@pytest.fixture
def PrivateIndexPipenvInstance(PipenvInstance):
    # This uses the real PyPI since we need Internet to access the Git
    # dependency anyway.
    with PipenvInstance() as p:
        with open(p.pipfile_path, 'w') as f:
             f.write(contents)
        yield p
```

that way you don't have to repeat the `@pytest.mark.install` as much and you can slim down the test to make it easier to read.

Reply:
```
def test_private_index_skip_lock(PrivateIndexPipenvInstance):
    p = PrivateIndexPipenvInstance
    c = p.pipenv('install --skip-lock')
    assert p.pipenv('install --skip-lock')
```

or if you want to keep the convention that you call PipenvInstances in a with statement, can do:

```
def PrivatePipenvIndexInstance(PipenvInstance):
     @contextlib.contextmanager
     def inner():
         with PipenvInstance() as p:
             # write stuff
             yield p
     return inner
```